Source/Manager/User/User.swift

```diff
@@ -225,15 +225,22 @@ public class User: UserProtocol {
 
         log(from: self, "new tokens \(newTokens)")
 
-        if let makePersistent = makePersistent {
-            self.isPersistent = makePersistent
+        self.isPersistent = makePersistent ?? self.isPersistent
+
+        let isNewLoggedInUser: Bool
+        if let newAnyUserID = newTokens.anyUserID, newAnyUserID != tokens.old?.anyUserID {
```

Comment:
Would `let isNewLoggedInUser = newTokens.anyUserID != nil && newTokens.anyUserID != tokens.old?.anyUserID` be clear enough?